src/drawer/story.tsx

```diff
@@ -84,38 +84,39 @@ class DrawerDemo extends Component {
     isOpen: false,
   }
 
-  onToggleDrawer = (open: boolean) => {
-    this.setState({ isOpen: open })
-  }
-
   toggleDrawer = () => {
-    if (this.drawer) {
-      this.drawer.close()
+    if (this.state.isOpen) {
```

Comment:
nit: you could just do a toggle

```
toggleDrawer = () => {
  this.setState(({ isOpen })=> ({ isOpen: !isOpen }))
}
```

Reply:
I had a typescript issue and I didn't really want to spend time investigating on this, so I merged it  :/